packages/store/src/module.ts

```diff
@@ -110,6 +92,45 @@ export class NgxsModule {
     };
   }
 
+  private static ngxsTokenProviders(
+    states: StateClass[],
+    options: NgxsModuleOptions
+  ): Provider[] {
+    return [
+      {
+        provide: NGXS_TEST_MODE,
+        useFactory: NgxsModule.isAngularInTestMode
+      },
+      {
+        provide: NG_TEST_MODE,
+        useFactory: NgxsModule.isAngularDevMode
```

Comment:
Surely this should be NG_DEV_MODE to match what it represents?

Reply:
done